plugins/lsp/language-servers/nil-ls-settings.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  helpers,
+}:
+# Options:
+#  - https://github.com/oxalica/nil/blob/main/docs/configuration.md?plain=1#
+with lib; {
+  formatting = {
+    command = helpers.defaultNullOpts.mkListOf types.str null ''
```

Comment:
`mkListOf str` produces a list of `maybeRaw str`, which is different from the `listOf str` we had before.

Would something like this be more correct @GaetanLepage?

```nix
mkNullable (listOf str) "null" "<description-here>"
```

Or should this type accept `__raw` lua strings @khaneliman?

Reply:
I believe it'd just be the executable path or name.

Reply:
Gonna be hitting the road for a few hours. If this needs to change I can address later.

Reply:
I'd like to run this by @traxys as his recent comments on one of my PRs are making me second guess this feedback.

Raw strings are really just lua code; i.e. no quotes. So a user might want to use a raw string here to reference some other lua string-variable defined elsewhere.

_I guess_ the only time we'd want to avoid them is when the value _won't_ end up directly in lua code, or we're going to do some string manipulation on it (and assume it is a string literal)